impeller/entity/contents/content_context.h

```diff
@@ -387,6 +394,16 @@ class ContentContext {
 
   bool IsValid() const;
 
+  /// This setting does two things:
+  /// 1. Enables clipping with the depth buffer, freeing up the stencil buffer.
+  ///    See also: https://github.com/flutter/flutter/issues/138460
+  /// 2. Switches the generic tessellation fallback to use stencil-then-cover.
+  ///    See also: https://github.com/flutter/flutter/issues/123671
+  ///
+  // TODO(bdero): Remove this setting once StC is fully de-risked
```

Comment:
I don't see an enumeration of anticipated risks. We should jot them down so we aren't surprised. Perhaps discuss during the weekly and I can prep. a list.

Reply:
I don't think we need to do that in the source code, we can handle that in the issue tracker

Reply:
Totally, just pointing out that we don't have that written down anywhere.

Reply:
Just to clarify my thinking here around this flag:

I think "fully de-risked" wasn't quite the right terminology for me to use here. Overall this is a big change to the perf characteristics of paths and clips, and the biggest risk in my eyes is users encountering slow (but reasonable) usage patterns that I haven't come across/considered.

After the flag flip, overall risk will rapidly decay through:
* Seeing benchmark & golden results
* The team churning on tasks at ToT
* The engine rolling into g3
* Smoke testing (Wondrous, Gallery)
* Profiling the trouble cases

I anticipate the flag's lifespan to be a few weeks. Maybe slightly longer if nontrivial problems pop up and the flag needs to get flipped back and forth a few times.

This flag impacts the config of most RenderPasses and draw calls, so if at some point we decide to keep it around for several months (which I doubt will happen), we should set up golden variations for both modes to avoid bitrot.